rls/src/test/java/io/grpc/rls/AdaptiveThrottlerTest.java

```diff
@@ -72,7 +74,7 @@ public void shouldThrottle() {
         .of(1.0f / 3.0f);
 
     // Skip half a second (half the duration).
-    fakeClock.forwardTime(500 - fakeClock.currentTimeMillis(), TimeUnit.MILLISECONDS);
+    fakeClock.forwardTime(500 - (fakeClock.currentTimeMillis() - startTime), TimeUnit.MILLISECONDS);
```

Comment:
Does it mean the original code had a bug and worked only because the epoch offset was small?

Also the comment should really say "Skip *to* half second mark from the beginning"  to make it easy to understand.

Reply:
Yeah, it was a bug in the test.